plugins/modules/mso_user.py

```diff
@@ -228,7 +228,6 @@ def main():
         payload = dict(
```

Comment:
Why do we have mso_user changes in this PR. It is in the other one right?
I am not reviewing those mso_user changes in this PR. Remove them and make sure they are in the other one.